packages/angular/projects/clr-angular/src/data/datagrid/providers/selection.ts

```diff
@@ -160,6 +160,8 @@ export class Selection<T = any> {
         this.lockedRefs = updateLockedRef;
       })
     );
+
+    this.subscriptions.push(this.valueCollector.pipe(debounceTime(0)).subscribe(() => this.emitChange()));
```

Comment:
Do [this](https://github.com/vmware/clarity/blob/next/packages/angular/projects/clr-angular/src/data/datagrid/providers/selection.spec.ts#L214) test cover this use case?

Reply:
The test is failing I am looking into fixing it.